scss/mixins/_buttons.scss

```diff
@@ -24,7 +24,7 @@
     @if $enable-shadows {
       box-shadow: $btn-box-shadow, 0 0 0 2px rgba($border, .5);
```

Comment:
Not too familiar with enable-shadows and the thought process behind this line but possibly $btn-box-shadow should be $btn-focus-box-shadow here as well.